ecosystem/sep-0024.md

```diff
@@ -402,6 +446,8 @@ Because `callback` is used to notify the wallet that the interactive flow is com
 Wallet applications display content rendered by a third party anchor service. It is recommended to use the [`rel=noopener`](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener) attribute on links or the `noopener` feature for [`Window.open()`](https://developer.mozilla.org/en-US/docs/Web/API/Window/open#window_functionality_features).
 
 Note that using `noopener` prohibits the use of `postMessage` callbacks. If `postMessage` callbacks are required for your implementation, it recommended to only open URLs from anchors that you, the wallet developer, trust.
+   
+To validate the provenance and integrity of the callback, anchors [must sign it and the UI must validate the signature](#callback-signature) before taking any action.
```

Comment:
For on_change_callbacks, the callbacks are sent after the interactive flows when there's an update to `statue` or `kyc_verified` props; for callbacks, it looks like the callbacks are also sent after the user finishes the interactive flow.

If I understand it correctly, we would want to take `UI` out from the sentence.
```suggestion
To validate the provenance and integrity of the callback, anchors [must sign it](#callback-signature) and wallet servers must validate the signature before taking any action.
```